ManagedClientConsoleAppSample/Program.cs

```diff
@@ -11,10 +12,11 @@ class Program
         //============= Config [Edit these with your settings] =====================
         internal const string vstsCollectionUrl = "https://myaccount.visualstudio.com"; //change to the URL of your VSTS account; NOTE: This must use HTTPS
         // internal const string vstsCollectioUrl = "http://myserver:8080/tfs/DefaultCollection" alternate URL for a TFS collection
+        internal const string clientId = "0fa17cf4-f75c-4185-ab9a-7c5ea47ca073"; //change to your app registration's Application ID
+        internal const string replyUri = "http://localhost:8080"; //change to your app registration's reply URI.
```

Comment:
Typically you use the base url of your app. Which for a single page app hosted at 'localhost:8080' will be 'localhost:8080'. Having the user set the replyUri to something like this is not unusual. In other ADAL.JS samples (https://github.com/Azure-Samples/active-directory-javascript-singlepageapp-dotnet-webapi) they instruct users to set the reply url to "localhost:44302"